src/libs/NextStepUtils.ts

```diff
@@ -159,15 +161,10 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
                     }
                 }
 
-                optimisticNextStep.message.push(
-                    {
-                        text: `automatically submit${harvestingSuffix}!`,
-                        type: 'strong',
-                    },
-                    {
-                        text: ' No further action required!',
-                    },
-                );
+                optimisticNextStep.message.push({
+                    text: `automatically submit${harvestingSuffix}`,
+                    type: 'strong',
+                });
             }
 
             // Prevented self submitting
```

Comment:
I have raised this in slack internally

Reply:
Summary @rayane-djouah @eVoloshchak we have decided to remove this next step from newDot, its should not be possible and showing this next step does not even make much sense because the user should tno even be able to submit in this case.

Lets remove it then